pkg/chartutil/create.go

```diff
@@ -147,16 +147,13 @@ service:
 
 ingress:
   enabled: false
+  # className: nginx
   annotations: {}
-    # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
-  hosts:
+  rules:
     - host: chart-example.local
-      paths:
-      - path: /
-        backend:
-          serviceName: chart-example.local
-          servicePort: 80
+      path: /
```

Comment:
I'm not sure removing support for multiple paths is a good idea, although I admit I rarely supply multiple paths. Let's hear other opinions as well.

Reply:
Unless this is something that was written up in a proposal and was generally accepted by the broader community, we cannot make significant changes in functionality to `helm create`'s output. Someone out there put multiple path support in here. We can't just remove that without justification.

Reply:
this path value changed not so lang ago and was breaking too...

not it is combined with the rule - especially since a "virtual host" usually is the hostname - and not does not include the path.

I have a limited understanding why to have the create command compatible to something helm specific and not adapt to changes at kubernetes itself... again, helm create is one-time command; In my eyes to ease accessibility for creating helm charts... it should be as easy as possible for 80% of the cases

Reply:
The changes in the new ingress object are not related to how Helm charts are structured, they are only semantic changes. I think the idea for keeping compatibility in this case is to keep supporting the same features as before, even on the newer ingress version. It's certainly possible to do that regardless of the breaking changes.

I also think this kind of change would require justification and I don't see any.